modelcontextprotocol/server.py

```diff
@@ -486,6 +490,191 @@ def update_assets_tool(
+            - name (str): Name of the glossary (required)
+            - description (str, optional): Short description of the glossary
+            - long_description (str, optional): Detailed description of the glossary
+            - certificate_status (str, optional): Certification status ("VERIFIED", "DRAFT", or "DEPRECATED")
+
+    Returns:
+        Dict[str, Any]: Dictionary containing:
+            - results: List of dictionaries for each glossary creation attempt with details:
+                - index: Index of the glossary in the input (0 for single glossary)
+                - guid: The GUID of the created glossary (if successful)
+                - name: The name of the glossary
+                - qualified_name: The qualified name of the created glossary (if successful)
+                - success: Boolean indicating if creation was successful
+                - errors: List of any errors encountered for this specific glossary
+            - overall_success: Boolean indicating if all glossaries were created successfully
+            - errors: List of overall errors (not specific to individual glossaries)
+
+    Examples:
+        # Create a single glossary
+        result = create_glossary_assets_tool({
+            "name": "Business Terms",
+            "description": "Common business terminology",
+            "certificate_status": "VERIFIED"
+        })
+
+        # Create multiple glossaries
+        NOTE-When someone asks to create multiple glossaries do it in a single call.
+        result = create_glossary_assets_tool([
+            {
+                "name": "Business Terms",
+                "description": "Common business terminology",
+                "certificate_status": "VERIFIED"
+            },
+            {
+                "name": "Technical Dictionary",
+                "description": "Technical terminology and definitions",
+                "certificate_status": "DRAFT"
+            }
+        ])
+    """
+
+    # Parse parameters to handle JSON strings using shared utility
+    try:
+        glossaries = parse_json_parameter(glossaries)
+    except json.JSONDecodeError as e:
+        return {"error": f"Invalid JSON format for glossaries parameter: {str(e)}"}
+
+    return create_glossary_assets(glossaries)
+
+
+@mcp.tool()
+def create_glossary_term_assets_tool(terms) -> dict[str, Any]:
+    """
+    Create one or multiple AtlasGlossaryTerm assets in Atlan.
+
+    Args:
+        terms: Either a single term specification (dict) or a list of term specifications.
+            Each specification can be a dictionary or GlossaryTermSpecification object containing:
+            - name (str): Name of the term (required)
+            - glossary_guid (str): GUID of the glossary this term belongs to (required)
+            - description (str, optional): Short description of the term
+            - long_description (str, optional): Detailed description of the term
```

Comment:
This needs to be changed to user_description as per the Glossary Pydantic Models defined